Tracing: Docs and config improvements for Tempo/Jaeger/Zipkin (#65255)
grafana/grafana: docs/sources/explore/trace-integration.md

> **Note:** Available in Grafana 7.4 and later versions.

You can navigate from a span in a trace view directly to logs relevant for that span. This is available for Tempo, Jaeger, and Zipkin data sources at this moment. Refer to their [relevant documentation](https://grafana.com/docs/grafana/latest/datasources/tempo/#trace-to-logs) for instructions on how to configure this feature.

{{< figure src="/static/img/docs/explore/trace-to-log-7-4.png" class="docs-image--no-shadow" max-width= "600px"  caption="Screenshot of the trace view in Explore with icon next to the spans" >}}

Click the document icon to open a split view in Explore with the configured data source and query relevant logs for the span.

### Trace to metrics

> **Note:** This feature is currently in beta & behind the `traceToMetrics` feature toggle.

> You can navigate from a span in a trace view directly to metrics relevant for that span. This is available for Tempo, Jaeger, and Zipkin data sources at this moment. Refer to their [relevant documentation](https://grafana.com/docs/grafana/latest/datasources/tempo/#trace-to-metrics) for instructions on how to configure this feature.

Reviewer: ```suggestion
You can navigate from a span in a trace view directly to metrics relevant for that span. This feature is available for Tempo, Jaeger, and Zipkin data sources. Refer to their [relevant documentation](https://grafana.com/docs/grafana/latest/datasources/tempo/#trace-to-metrics) for configuration instructions.
```

## Service Graph view

The Service Graph view visualizes the span metrics (traces data for rates, error rates, and durations (RED)) and service graphs.
Once the requirements are set up, this pre-configured view is immediately available.

> For more information, refer to the [Service Graph view section](https://grafana.com/docs/grafana/latest/datasources/tempo/#open-the-service-graph-view) of the Tempo data source page and the [service graph view page](https://grafana.com/docs/tempo/latest/metrics-generator/service-graph-view/) in the Tempo documentation.

Reviewer: ```suggestion
For more information, refer to the [Service Graph view section]({{< relref "/docs/grafana/latest/datasources/tempo/#open-the-service-graph-view" >}}) of the Tempo data source page and the [service graph view page]({{< relref "/docs/tempo/latest/metrics-generator/service-graph-view/" >}}) in the Tempo documentation.
```
